.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
I'm using vim. I did a research and I found Vexplore, and it's works well for me. Can you try on your computer?